microservices/nris_api/postgres/Dockerfile.dev

```diff
@@ -0,0 +1,7 @@
+FROM postgres:9.6
+
+# Copy only migration scripts for creation
+RUN echo "CREATE USER postgres" >> /docker-entrypoint-initdb.d/01-create-user.sql
```

Comment:
How/when/where is this directory being created? And is this command just meant to create & fill that sql file with the command string?

If so, why not just have that file COPY'd with the content already in it? That seems like the simpler solution to me.

Reply:
the postgres s2i looks for that specific files and runs it i think? and we use env variables to change the superuser, but use this to still make sure there is a `postgres` user in the db.... i think that's right. @FreshGyan?

Reply:
This is only for local development so s2i hasnt played a role yet.

However this was added because by default the docker image that comes from dockerhub creates the superuser with whatever username you provide. i.e. mds
But the docker image that comes from redhat already comes with a superuser named postgres.

The flyway migrations use the postgres user and thus that user is just created so that locally we dont have failing migrations.



The reason i went with just echoing the contents instead of COPY is because then i'd have to grab a copy of the entrypoint file that's supplied by postgres and then add my command and overwrite the one in the container and that just seemed like too much work to last year me 😅